packages/core/package.json

```diff
@@ -88,7 +88,7 @@
     "dependencies": {
         "@babel/polyfill": "^7.4.4",
         "@deriv/account": "^1.0.0",
-        "@deriv/analytics": "^1.0.0",
+        "@deriv/analytics": "^1.1.0",
```

Comment:
You may need to run `npm run bootstrap:dev` for updating the `package-lock` file :thinking:

Reply:
its not the latest version. isn't @amir-deriv ?